src/js/contoller.js

```diff
@@ -0,0 +1,110 @@
+import { render, countItem } from "./view.js"
+import { saveTodo } from "./service.js"
+import { data, setLocalStorage } from "./model.js"
+
+
+export const handleAddTodo = (event) => {
+  event.preventDefault()
+
+  const newTodo = document.querySelector("#new-todo-title").value
+  if (newTodo === '') return
+
+  document.querySelector("#new-todo-title").value = ''
+
+  saveTodo(newTodo)
+  setLocalStorage()
+  render()
+}
+
+export const handleCheckButton = (event) => {
+  const checkBox = event.target.closest("input[type=checkbox]")
+  if (checkBox === null) return
+  for (const item of data.todos) {
+    if (item.id === +checkBox.id) {
```

Comment:
```suggestion
    if (item.id === Number(checkBox.id)) {
```

요런식으로 숫자타입으로 변환한다는 것을 명시적으로 나타내는게 가독성에 더 좋을듯